.circleci/config.yml

```diff
@@ -80,11 +80,20 @@ workflows:
   version: 2
   build-and-test:
     jobs:
-      - build
-      - test
+      - build:
```

Comment:
Is it possible to apply a filter to an entire workflow?

Reply:
Nvm, doesn't appear to be. :-1: CircleCi

Reply:
I was looking for that too. Circle CI doesn't seem to support that.